iotilecore/iotile/core/hw/debug/flash_board_step.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import (unicode_literals, print_function, absolute_import)
+from builtins import str
+from iotile.core.hw.hwmanager import HardwareManager
+from iotile.core.exceptions import ArgumentError
+
+class FlashBoardStep (object):
+    """A Recipe Step used to flash firmware directly
+
+    Currentlly only supports using jlink to flash. Meant to bootstrap firmware
+
+    Args:
+        addresses (list[int]): List of slot addresses to check against
```

Comment:
These don't seem to be the arguments that your class takes..  Docstrings are not the place to be aspirational about future capabilities.